sale_automatic_workflow/models/sale_order.py

```diff
@@ -62,7 +62,7 @@ def _onchange_workflow_process_id(self):
             self.picking_policy = workflow.picking_policy
         if workflow.team_id:
             self.team_id = workflow.team_id.id
-        if workflow.warning:
+        if workflow.warning and workflow.validate_order:
```

Comment:
@SodexisTeam I think this is done at purpose following help message:

![image](https://user-images.githubusercontent.com/19529533/233022759-351de06f-4423-4511-a832-a3b2e65ca3f3.png)

So, this change is not wanted.